tests/ttnn/sweep_tests/sweeps/sweeps/multigammaln.py

```diff
@@ -16,10 +16,10 @@
     "batch_sizes": [(1,)],
     "height": [384, 1024],
     "width": [1024, 4096],
-    "input_dtype": [ttnn.bfloat16],
+    "input_dtype": [ttnn.bfloat16, ttnn.bfloat16],
```

Comment:
```
def skip(
    batch_sizes,
    height,
    width,
    input_dtype,
    input_memory_config,
    output_memory_config,
    layout,
) -> Tuple[bool, Optional[str]]:
    if layout == ttnn.ROW_MAJOR_LAYOUT:
        return True, "Skipped as ROW_MAJOR_LAYOUT not supported"
    return False, None

```
If RM is not supported, can you skip the test for the same?

Reply:
Changed @Aswinmcw